src/frontend/hooks/server/projects.ts

```diff
@@ -147,3 +148,14 @@ export function useImportProjectConfig() {
     },
   });
 }
+
+export function useGetOwnRole() {
+  const {projectId, projectApi} = useActiveProject();
+
+  return useQuery({
+    queryKey: [OWN_ROLE, projectId],
```

Comment:
`OWN_ROLE` is not fully necessary at this point, but in the future there could be the possibility of your role changing, hence why I created this query key